testsuite/in_prefix/README.md

```diff
@@ -0,0 +1,157 @@
+# "in-prefix" compiler testing
+
+This directory performs tests on the compiler after it has been installed to the
+prefix given to `configure`. The test requires at least the Unix library to have
+been compiled, and only works if bindir and libdir share a common root directory
+(e.g. /usr/bin and /usr/lib/ocaml share a common root of /usr, but /bin and
+/lib/ocaml do not share a common root directory). The test needs to be able to
+rename this common root directory by adding a suffix ".new". For this reasoa,
```

Comment:
```suggestion
(e.g. `/usr/bin` and `/usr/lib/ocaml` share a common root of `/usr`, but `/bin` and
`/lib/ocaml` do not share a common root directory). The test needs to be able to
rename this common root directory by adding a suffix ".new". For this reason,
```

Reply:
I had a thought (just to explain it), that putting these paths as `code` wasn't aiding readability (either rendered or as text) - i.e. I was finding "/usr/bin and /usr/lib/ocaml" easier to read (because each path is "introduced" by a slash anyway) than the version where the backticks are kinda getting in the way?

Reply:
fine by me :)